### DISCUSSION TITLE
Simple redirection service for naked domains

### DISCUSSION THREAD

**Discussion by egohookx** [beginner-corp/begin-community#84] [Ideas]
This might be overkill, but for those wanting to use naked domains (something.com), and who don't have the ability to set A alias records with their DNS provider, could a redirector be set up so that we can create A records to an IP and allow HTTP[S] redirection to our apps? Extreme example: http://something.com/foo?bar=duh -> https://app.something.com/foo?bar=duh

In my case, Route53 isn't letting me set up an alias A record in the way I'd expect (despite the Begin docs suggesting it's supported) so I'm forced to use an external redirector. In my case I went with redirect.pizza (seriously).

**Comment by brianleroux** [comment]
hi @egohookx, very sorry about that experience! route53 should work and if not that's a bug. I'd love to have a look at this with you if you can spare the time. my calendar https://calendly.com/brianleroux (if not no problem; redirect def works and thanks for reporting!)
